src/abstractmcmc.jl

```diff
@@ -224,6 +202,78 @@ function AbstractMCMC.step(
     return Transition(t.z, tstat), newstate
 end
 
+##############
+### Turing ###
+##############
+
+function AbstractMCMC.step(
+    rng::AbstractRNG,
+    model::LogDensityModel,
+    spl::AbstractMCMC.AbstractSampler;
+    init_params = nothing,
+    kwargs...,
+)
```

Comment:
This definition is too generic, isn't it? We surely don't want this method be hit for every sampler, RNG, and log density model.